changelog.d/5-internal/pr-2699

```diff
@@ -0,0 +1 @@
+The `POST /delete` endpoint of the account API is now migrated to servant
```

Comment:
we should probably condense the changelog a little once all these small PRs are merged?